test/config.sh

```diff
@@ -214,5 +215,47 @@ collect_coverage() {
     mv cover.router.prof $(mktemp -u ./coverage/integration.XXXXXXXX) || true
 }
 
+################################
+# performance utils
+################################
```

Comment:
As these are only used by one test, can we put them in that test?  config.sh is getting too big.